packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.87.1
+------------------------------
+*April 12, 2021*
+
+### Changed
+- `checkoutTimeout` prop now defaults to ten seconds
```

Comment:
Out of interest – why is the timeout so high now? 10 seconds seems a long time to expect users to hang around for if data hasn't loaded